### ISSUE TITLE
Linux 18 Build fail - /v8/v8engine.h:10:10: fatal error: v8.h: No such file or directory  #include <v8.h>

### ISSUE THREAD

**Issue by inzider** [MultiChain/multichain#103]
Hi, 

I'm trying to build the multichain source code but get this error. 

"/v8/v8engine.h:10:10: fatal error: v8.h: No such file or directory
 #include <v8.h>".

All files are in place. I can see the V8.h file in the folder as referenced in V8engine.h.
Please confirm where the V8 build should go since the "extract in current folder" in instructions is not clear.  

Any advice ? 

Thanks in advance.

**Comment by khalidmaquilang** [comment]
im currently encountering this issue.. it is almost 6years.. any update?

**Comment by gidgreen** [comment]
According to the instructions at the link below, you should put these files in the `v8build` directory that you create.

https://github.com/MultiChain/multichain

**Comment by khalidmaquilang** [comment]
Yes you are right.. it will automatically detect and there is also a problem during compilation but i fixed it.. if i have time, i will create a PR

> According to the instructions at the link below, you should put these files in the `v8build` directory that you create.
> 
> https://github.com/MultiChain/multichain
